### DISCUSSION TITLE
【live2d 】Cubism3后续有支持的计划吗？

### DISCUSSION THREAD

**Discussion by tiandankanfeng** [tangly1024/NotionNext#2079] [Ideas]
**为什么提出这个新的特性改动**
Cubism3下的看板娘形象以及交互更加丰富些

**描述一下你推荐的解决方案**
简要说明你的解决方案建议，例如，『Giscus评论插件功能更加强大，用户只需留言既可在你的邮箱收到通知。。。』

**描述一下你考虑过的其它替代解决方案**
简要说明你所有想过的有可能解决此问题的方案。

**补充说明**
补充与此特性相关的内容

**Comment by tangly1024** [comment]
未来是有可能的
